pkg/platform/model/svc.go

```diff
@@ -24,11 +24,28 @@ func NewSvcModel(ctx context.Context, cfg *config.Instance) (*SvcModel, error) {
 	return &SvcModel{ctx, client}, nil
 }
 
+type versionResponse struct {
+	Version graph.Version `json:"version"`
+}
```

Comment:
Let's put this in the `graph` package in a file that we maintain ourselves. Only the one file in there is generated, the package itself is open for us to add our own stuff to.